packages/agent/src/invitation/transactions.ts

```diff
@@ -0,0 +1,28 @@
+import type { CredentialsForProofRequest } from './handler'
+
+export type FormattedTransactionData = ReturnType<typeof getFormattedTransactionData>
+export type QtspInfo = CredentialsForProofRequest['verifier']
+
+export const getFormattedTransactionData = (credentialsForRequest?: CredentialsForProofRequest) => {
+  if (!credentialsForRequest) return undefined
+
+  const transactionData = credentialsForRequest.transactionData
+
+  if (!transactionData || transactionData.length === 0) return undefined
+
+  // Only allow one transaction data entry
+  if (transactionData.length < 1) throw new Error('Multiple transactions are not supported yet.')
```

Comment:
I guess should be `>` instead of `<`?

```suggestion
  if (transactionData.length > 1) throw new Error('Multiple transactions are not supported yet.')
```